### ISSUE TITLE
Implement Specialty Skill in hero detail page

### ISSUE THREAD

**Issue by RaphaelDDL** [EpicSevenDB/ui#10]
We already have the data on database (`specialtySkill`), just need to create a new section to show it. Example data:

```
    "specialtySkill": {
        "name": "Competitive Noble",
        "description": "After leaving her family mansion, she hopes to learn much about the outside world.",
        "dispatch": "[Explore] type",
        "enhancement": "Reward bonus +6%",
        "stats": {
            "command": 80,
            "charm": 88,
            "politics": 52
        }
    },
```

Requisite: Need to think of a smart way of assigning the specialty skill image to the hero. The icons aren't 1:1 with the heroes, some are repeated.

Or I just straight out duplicate the icons by adding to the hero folder of the assets, like I did with hero's normal skills and images

**Comment by RaphaelDDL** [comment]
Adding as High. We have this data since day 1 and I've been postponing since the icons are not per-hero, so I'd have to manually sort them out. But we could release a MVP iconless.

**Comment by RaphaelDDL** [comment]
Implemented on 1.1.4 a26872f1fb1db0661f76ea92be09a0ffb1601d4b

Images still pending, but they repeat a lot, so still gotta think on how implement properly
